polaris-react/src/components/Autocomplete/README.md

```diff
@@ -146,8 +146,14 @@ function MultiAutocompleteExample() {
     [deselectedOptions],
   );
 
+  const handleSelect = useCallback((selected) => {
+    setSelectedOptions(selected);
+    updateText('');
+  }, []);
+
   const removeTag = useCallback(
-    (tag) => () => {
+    (tag) => (event) => {
+      event.stopPropagation();
```

Comment:
Click events inside of `verticalContent` need to be prevented from bubbling up so that they don't trigger input focus.